packages/mermaid/src/diagrams/flowchart/types.ts

```diff
@@ -11,6 +11,12 @@ export interface FlowVertex {
   styles: string[];
   text?: string;
   type?: string;
+  icon?: string;
+  form?: string;
+  pos?: 't' | 'b';
+  img?: string;
+  assetWidth?: number;
+  assetHeight?: number;
```

Comment:
We should define these in only one place, and inherit it here.

Reply:
Will do